backend/drm/backend.c

```diff
@@ -11,7 +11,7 @@
 #include <wlr/interfaces/wlr_output.h>
 #include <wlr/types/wlr_list.h>
 #include <wlr/util/log.h>
-#include <wlr/util/signal.h>
+#include <util/signal.h>
```

Comment:
single quotes for internal headers

Reply:
Also, I'm kind of alphebetizing these lately

Reply:
Okay, maybe we should unify headers. What about sorting but keeping external and internal includes groupped?

Reply:
Yeah, that.